tests/slr_jest/admin/layouts/actions.spec.js

```diff
@@ -35,6 +50,12 @@ function getTestDocument() {
 						>
 							Edit
 						</button>
+						<button
```

Comment:
While this `getTestDocument` function is pre-existing, it should be removed in favour of using the [general one defined in the bootstrap file](https://github.com/the-events-calendar/event-tickets/blob/652afaef7122a0b937b61bb0c6f6bf82ac049bea/tests/slr_jest/_bootstrap.js#L173) and pointing it to a PHP snapshot (e.g. `tests/slr_integration/Admin/__snapshots__/Maps_Layout_Homepage_Test__test_layouts_tab_card_listing__0.snapshot.html`).

The problem the general `getTestDocument` is trying to solve is the one where the JS will be exercised on the actual PHP output, not a synthetic DOM made up for the tests.